index.mdx

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
Hi @ekaonthenet, thank you for your contribution. 

May I ask what is this file about ?

And also please refrain from committing yarn lock file to the repo. Thank You.

Reply:
My bad, that file should not be here, wrong drag n drop.
Removing it and `yarn.lock` now.